### ISSUE TITLE
How to batch processing with BoudingBoxes (also sequential option possible?)

### ISSUE THREAD

**Issue by mvanlierBCG** [aleju/imgaug#649]

**Comment by aleju** [comment]
Not sure what you mean with the question. There is a documentation page about BB augmentation, see https://imgaug.readthedocs.io/en/latest/source/examples_bounding_boxes.html . There is also a jupyter example notebook at https://nbviewer.jupyter.org/github/aleju/imgaug-doc/blob/master/notebooks/B02%20-%20Augment%20Bounding%20Boxes.ipynb . Maybe these help?

**Comment by mvanlierBCG** [comment]
No this is not exactly what I mean.

In notebook at [https://nbviewer.jupyter.org/github/aleju/imgaug-doc/blob/master/notebooks/A01%20-%20Load%20and%20Augment%20an%20Image.ipynb](url), you process multiple images (`images = [image, image, image, image]`) with iaa.Sequential. 

But how can I process the bounding boxes of these (multiple) images? like is done in the notebook [https://nbviewer.jupyter.org/github/aleju/imgaug-doc/blob/master/notebooks/B02%20-%20Augment%20Bounding%20Boxes.ipynb](url), but with multiple bounding boxes, with multiple images? Since that notebook provides only example for one `BoundingBoxesOnImage`.

Would that look in dummy code like this:
`
images = [image0, image1, image2]
bbss = [BoundingBoxesOnImage(), BoundingBoxesOnImage(), BoundingBoxesOnImage()]

image_aug, bbs_aug = seq(image=image, bounding_boxes=bbss)
`

Would that be possible? Or what are the other options?

**Comment by aleju** [comment]
Yes, if you have a batch of `N` images and `N` lists of bounding boxes, you can convert each BB-list to a `BoundingBoxesOnImage` instance and call `images_aug, bbsoi_aug = seq(images=<image list>, bounding_boxes=<BBsOI list>)`.

You can also provide the bounding boxes as a list of list of `BoundingBox` (i.e. one BB-list per image) or a list of `(N, 4)` numpy arrays (one BBs-array per image) or a list of list of `(4,)` numpy arrays (one list per image and one array per BB), it's not strictly required to go through `BoundingBoxesOnImage` or `BoundingBox`.
